src/App/ActivityList.purs

```diff
@@ -123,7 +127,26 @@ panelsListView panel =
 
 listView :: forall cs m. Array Todo -> HH.HTML cs m
 listView todos =
-  HH.ul_ (map (\todo -> HH.li [] [HH.text  todo.name]) todos)
+  HH.div [ Prop.class_ (ClassName "itemContainer")] (map todoView  todos)
+
+todoView :: forall cs m. Todo -> HH.HTML cs m 
+todoView todo =
+  HH.li 
+    [ Prop.class_ $ ClassName "item"
+    , Prop.attr (AttrName "style")  $ "background-color: " <> priorityToColor todo.priority
```

Comment:
It will be nice to eventually move styling to a css file.

Reply:
@tmciver  We definitely need to discuss this. I currently have a big stylesheet called `index.css` in the `dev` folder. At the same time inline css styling is also cool. 

There is this typed css library called purescript-css that could also be interesting for us.